scripts/20PP-JP/c100260017.lua

```diff
@@ -13,7 +13,7 @@ function c100260017.initial_effect(c)
 	c:RegisterEffect(e1)
 end
 function c100260017.filter1(c,e)
-	return c:IsType(TYPE_PENDULUM) and c:IsCanBeFusionMaterial() and c:IsAbleToRemove()
+	return c:IsType(TYPE_PENDULUM) and c:IsCanBeFusionMaterial() and c:IsAbleToRemove() and (not e or not c:IsImmuneToEffect(e))
```

Comment:
why?

Reply:
> why?

Is the check unnecessary?

Reply:
the card(s) having immune effect in grave don't exist.